src/theory/arith/congruence_manager.h

```diff
@@ -101,6 +102,16 @@ class ArithCongruenceManager {
 
   eq::EqualityEngine d_ee;
 
+  /** proof manager */
+  ProofNodeManager* d_pnm;
+  // A proof generator for storing proofs of facts that are asserted to the EQ
```

Comment:
I believe we always use comments enclosed by `/** */` for documentation in h files.

Reply:
Done in 8e836b883.